utils/python/cime_util.py

```diff
@@ -493,16 +494,27 @@ def delete_jobs(jobs):
     del_cmd = "%s %s" % (BATCH_INFO[batch_system][1], " ".join(jobs))
     return run_cmd(del_cmd, ok_to_fail=True, verbose=True)
 
+
+###############################################################################
+def set_model(model):
+    global _MODEL
+    _MODEL = model
+
+def get_model():
+    global _MODEL
+    return _MODEL
+
 ###############################################################################
 def parse_config_machines():
 ###############################################################################
     """
     """
     global _MACHINE_INFO
+    model = get_model()
     if (_MACHINE_INFO is None):
         _MACHINE_INFO = {}
         import xml.etree.ElementTree as ET
-        config_machines_xml = os.path.join(get_cime_root(), "cime_config", "acme", "machines", "config_machines.xml")
```

Comment:
I encapsulated the retrieval of the config directory for a model but forgot to use it here, I'll fix that.